rpdc.py

```diff
@@ -520,6 +524,8 @@ def validateJSONConfigContent(variantConfig):
                 resultRapidModelID = generateOptimizedVariant(modelID, outputModelFilePrefix, variant, accessToken, baseUrl)
                 if (resultRapidModelID != -1):
                     newRapidModelIDs.append(resultRapidModelID)
+                else:
+                    failedOptimizations += 1
```

Comment:
Shouldn't the exit() happen on the first error?
Or is the idea to return an error code if some error occurred in the middle?

Reply:
Yea, its also possible to exit on the first error. The idea was when a user optimizes multiple files and one in the middle fails, it should trigger the full error at the end instead of prematurely